### PR TITLE
viro 1.0.0 new formula

### PR DESCRIPTION
**Pull request by LeonardoJPerez**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/viro.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+class Viro < Formula
+  desc "Command-line utility for securely managing environment variables"
+  homepage "https://meshstudio.io/"
+  url "https://s3.amazonaws.com/mesh-homebrew/viro"
```
Comment (fxcoudert) [inline Formula/viro.rb]: URL says “access denied”

File: Formula/viro.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  version "1.0.0"
+  sha256 "eb22f144fe249e45332edca2a05bb40eccc81a4013b0b555915af74c1345e601"
+
+  def install
+    bin.install "viro"
```
Comment (fxcoudert) [inline Formula/viro.rb]: Is this a single script?

File: Formula/viro.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  end
+
+  test do
+    system bin/"viro", "--version"
```
Comment (fxcoudert) [inline Formula/viro.rb]: We need a test that exercises the some of the functionality of the app.

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/viro.rb (hunk 1 above)
Comment (LeonardoJPerez) [reply Formula/viro.rb]: Oops, thanks!

File: Formula/viro.rb (hunk 2 above)
Comment (LeonardoJPerez) [reply Formula/viro.rb]: Yes.

**Comment by fxcoudert** [comment]
```
$ file ~/Downloads/viro 
/Users/fx/Downloads/viro: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), statically linked, with debug_info, not stripped
```

This looks like a Linux binary. Homebrew is for macOS software, that builds from source. I am closing the pull request for now, please open a new one once you have a formula that builds from source.

**Pull request closed without merging**